Docs might need tune up for people to get started to build docs

https://github.com/linkml/linkml/blob/main/docs/README.md just says

```markdown
To build the doc locally:

poetry run make html
```

but doesn't work for me neither from root, nor from `docs/`:

```shell
❯ pwd
/home/yoh/proj/misc/linkml/linkml

❯ git describe --tags
v1.6.11

❯ poetry run make html

Skipping virtualenv creation, as specified in config file.
make: *** No rule to make target 'html'.  Stop.

❯ (cd docs; poetry run make html; )
Makefile   code.rst  contributing/  developers/   examples.md  generators/   howtos/  index.rst  linkml.generators.rst  linkml.rst        links.rst  modules.rst  specifications/
README.md  conf.py   data/          ecosystem.md  faq/         glossary.rst  images/  intro/     linkml.reporting.rst   linkml.utils.rst  make.bat   schemas/     supplement-progress.md
Skipping virtualenv creation, as specified in config file.
Running Sphinx v7.2.6

Extension error:
Could not import extension sphinx_click (exception: No module named 'sphinx_click')
make: *** [Makefile:20: html] Error 2
```

--- a/docs/README.md
+++ b/docs/README.md
@@ -11,9 +11,11 @@ We use the sphinx framework.
 
 ## Instructions
 
-To build the doc locally:
+To build the docs locally, first make sure you have the poetry dev dependencies installed, then use poetry to run make html (don't run "make html" directly):
 
 ```bash
+poetry install --with dev
+cd docs
 poetry run make html
 ```
 

